runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -547,6 +552,22 @@ ClientSessionData::clearCaches()
       }
 
    _wellKnownClasses.clear();
+   setCachesAreCleared(true);
```

Comment:
This line is the same as `_requestUnloadedClasses = true;` which is found earlier in this method. If you add this, let's delete that line, 511.